drivers/gpio/gpio_shakti.c

```diff
@@ -1,3 +1,33 @@
+/****************************************************************************
+ * Project                               : Secure IoT SoC
+ * Name of the file                      : gpio_shakthi.c
+ * Brief Description of file             : This is a ZEPHYR OS GPIO Driver file for Mindgrove Silicon's GPIO Peripheral.
```

Comment:
The Description of the file must be inside the asterisks. Maximum length in a line is 100 columns.

Reply:
Done